lib/Common/Exceptions/ExceptionCheck.cpp

```diff
@@ -58,7 +58,8 @@ void ExceptionCheck::SetHandledExceptionType(ExceptionType e)
 #if DBG
     if(!(e == ExceptionType_None ||
          e == ExceptionType_DisableCheck ||
-         !JsUtil::ExternalApi::IsScriptActiveOnCurrentThreadContext() ||
+         (e & ExceptionType_OutOfMemory) == ExceptionType_OutOfMemory ||  // InitializeModuleRecord handles OOM during dynamic import
```

Comment:
Removed along with nested EnterScript.

---
In reply to: [114929334](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r114929334) [](ancestors = 114929334)